data/migrations/20190719081542_projects.js

```diff
@@ -0,0 +1,67 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('projects', tbl => {
+        tbl.increments()
+  
+        tbl
+          .string('project_name')
+          .notNullable()
+          .unique()
+
+        tbl
+          .string('project_description')
+          .notNullable()   
+    })
+
+    .createTable('actions', tbl => {
+        tbl.increments()
+        
+        tbl
+            .string('action_description')
+            .notNullable()
+    
+        tbl
+            .string('action_notes')
+            .notNullable()
+
+        tbl
```

Comment:
In the case of a one to many relationship, where an action is unique to a project, you would use this foreign key. In the case of the many to many relationship, where an action can be used with multiple projects, you would not be using this foreign key.